app/src/store/api/rest/types.ts

```diff
@@ -139,7 +145,15 @@ export interface RestSuccess<
 }
 
 /**
- * Rest success action payload
+ * Stats response with guildId
+ */
+export interface StatsSuccess {
```

Comment:
Do you still need this action or can you use the RestSuccess action (with routeArgs)?